auth.js

```diff
@@ -0,0 +1,95 @@
+const config = require('./config')
```

Comment:
putting files in the root is OK for one file but we should probably move all these new files into a `src/` folder to keep things organized